tools/extra/qpafilter/qpafilter.md

```diff
@@ -0,0 +1,152 @@
+# qpafilter #
+
+## SYNOPSIS ##
+```console
+qpafilter [-h] [-v] [-s SENSOR_MAP] {create,dump} ...
+```
+
+## DESCRIPTION ##
+
```

Comment:
How about adding a high-level summary of that qpafilter is?
```
qpafilter is a tool that, given critical sensor values as determined by Quartus Power Analyzer,
creates a binary file that can be used by the FPGA BMC (Board Management Controller) firmware to set
sensor thresholds. In addition to creating this binary file, qpafilter can parse it and dump it in a human
readable format.
```


Then after this, maybe put a subcommands section?
```
## SUB-COMMANDS ##
    create
        Transform qpa text file into a binary format for the BMC FW.

    dump
        Parse an existing binary file and dump in a human-readable format.
```